src/compas/utilities/itertools.py

```diff
@@ -185,10 +189,79 @@ def linspace(start, stop, num=50):
 
 
 def flatten(list_of_lists):
-    """Flatten one level of nesting"""
+    """Flatten one level of nesting.
+
+    Examples
+    --------
+    >>> a = [[1, 2, 3], [4, 5, 6]]
+    >>> list(flatten(a))
+    [1, 2, 3, 4, 5, 6]
+    """
     return chain.from_iterable(list_of_lists)
 
 
+def unflatten(lst, num):
```

Comment:
This function seems to be a limited version of `reshape`, that is, `unflatten(l, num) == reshape(l, (len(l)//num), num))`. I would suggest for the sake of not cluttering the API that this should be removed.

Reply:
👍